tests/cephfs/cephfs_upgrade/cephfs_post_upgrade_validation.py

```diff
@@ -0,0 +1,191 @@
+import json
+import random
+import string
+import time
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from tests.cephfs.cephfs_volume_management import wait_for_process
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def nfs_test(nfs_req_params):
+    """
+    NFS post upgrade validation
+
+    Args:
+        nfs_req_params: mandatory params for testing in below format
```

Comment:
please add parameters type to be passed while using the function like string, list etc as standard practice.